src/App.tsx

```diff
@@ -0,0 +1,118 @@
+// tailwind.config.js must have your paths set properly.
+
+import { useEffect, useState } from "react";
+import Papa from "papaparse";
+import MainDashboard from "./components/MainDashboard";
+import PlatformDashboard from "./components/PlatformDashboard";
+import type { Student } from "./components/types";
+import './index.css';
+
+function App() {
+  const [students, setStudents] = useState<Student[]>([]);
+  const [activeView, setActiveView] = useState<'main' | 'platform' | 'loop' | 'github'>('main');
+
+  const fetchCSV = async () => {
```

Comment:
```suggestion
  const fetchCsv = async () => {
```

always follow casing rules even if you are dealing with abbreviations.